Source/error.cpp

```diff
@@ -35,8 +35,8 @@ void InitNextLines()
 	char tempstr[1536]; // Longest test is about 768 chars * 2 for unicode
 	strcpy(tempstr, message.data());
 
-	WordWrapString(tempstr, LineWidth, GameFont12, 1);
-	const string_view paragraphs = tempstr;
+	const std::string wrapped = WordWrapString(tempstr, LineWidth, GameFont12, 1);
+	const string_view paragraphs = wrapped;

```

Comment:
Is there a need for a conversion to string_view here?

Reply:
Yes, so that the `substring` call below is zero-copy.

Reply:
Although `TextLines` is an array of `std::string`, so it doesn't matter

Reply:
Done